src/newcharacter.cpp

```diff
@@ -708,8 +710,9 @@ void draw_points( WINDOW *w, points_left &points, int netPointCost )
 {
     mvwprintz( w, 3, 2, c_black, clear_str );
     std::string points_msg = points.to_string();
-    int pMsg_length = utf8_width( points_msg );
-    mvwprintz( w, 3, 2, c_ltgray, points_msg.c_str() );
+    int pMsg_length = utf8_width( points_msg, true );
+    nc_color color;
```

Comment:
This needs to be initialized, probably just to `c_ltgray` and than you can use it as second color parameter in the function call below (as in `print_colored_text( ..., color, color, points_msg )`).